src/helm/common/critique_request.py

```diff
@@ -34,6 +34,11 @@ class CritiqueQuestionTemplate:
 
     Can contain placeholders like {{placeholder}} that will be interpolated using the fields in CritiqueRequest."""
 
+    image_url: Optional[str] = None
```

Comment:
I think this should be `media_object: Optional[MediaObject]` to make it general to other modalities.